3rdparty/libcxxrt/CMakeLists.txt

```diff
@@ -18,6 +18,11 @@ add_library(libcxxrt OBJECT
 set_target_properties(libcxxrt PROPERTIES
   POSITION_INDEPENDENT_CODE ON)
 
+# Configure clangw before turning off warnings.
+if (USE_CLANGW)
```

Comment:
As for the rest of the name, I'm still not sure. The problem is that `clang` (and `CLANGW`) is too general. We're trying to convey three distinct bits of information:

1. This is on Windows, but not MSVC nor Clang-for-Windows
2. This is going to cross-compile using Clang
3. Specifically this is going to compile ELFs (not PEs)

So `USE_PORTABLE_ENCLAVES` (our original term) is out because it abbreviates to PE which is overloaded and means the _opposite_ of what we're doing.

What about `BUILD_WINDOWS_ELF`? Ugh I don't like that either. Bruce suggested in scrum that we have a n naming contest 😄

Reply:
Can you look at the latest iteration of the PR and come up with a nice name?